docs/dev/project_management.rst

```diff
@@ -19,23 +19,29 @@ them easily.
 Our GitHub Workflow
 -------------------------------------------------------------------------------
 
-* We have 2 persistent branches: **main** and **dev**.
-* We create temporary feature branches off of **dev** and make pull requests to
-  **dev** throughout our 2 week long sprints.
-* At the end of each sprint, or any time a particularly significant feature has
-  been merged in and the nightly builds are passing, **dev** is merged into **main**.
+* We have 3 persistent branches: ``main`` (the default branch), ``nightly``, and
+  ``stable``.
+* We create temporary feature branches off of ``main`` and make pull requests into
+  ``main`` throughout our 2 week long sprints. All code that's merged into ``main``
+  should have passed our CI tests and been reviewed by at least one other person.
+* Every night the ``main`` branch is used to run the :ref:`nightly-data-builds`. If the
+  builds are successful, then the ``nightly`` branch is automatically updated to point
+  to the latest commit on ``main``. If the builds fail, then the ``nightly`` branch is
```

Comment:
Will `nightly` have branch protections so only a VM can update it?

Reply:
Just added these for `nightly` and `stable`: 
<img width="808" alt="image" src="https://github.com/catalyst-cooperative/pudl/assets/2495794/fdcb059a-bb04-44bf-96c2-388acbd81185">